lib/fake_artist/table.ex

```diff
@@ -346,20 +398,20 @@ defmodule FakeArtist.Table do
     Enum.count(state.players, fn {_k, v} -> v.voted_for == nil end) - game_master_vote == 0
   end
 
-  @spec get_winner(map()) :: binary()
-  defp get_winner(players) do
+  @spec is_trickster_picked(map()) :: boolean()
+  defp is_trickster_picked(players) do
```

Comment:
btw Elixir allows `?` as part of function names. So you can make this `is_trickster_picked?` or `trickster_picked?` for extra flavor if you want.

In fact I think that's a typical convention to show that something returns a Boolean value

Reply:
Had forgotten about that!